en/assembly/quick_start_cuav_v5_nano.md

```diff
@@ -79,17 +79,15 @@ If using a Spektrum satellite receiver, this should be connected to the flight c
 
 ## Power
 
-The *v5 nano* kit includes the *HV\_PM* module, which supports 2~10S LiPo batteries.
+The *v5 nano* kit includes the *HV\_PM* module, which supports 2~14S LiPo batteries.
 Connect the 6pin connector of the *HW\_PM* module to the flight control `Power` interface.
 
-> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_nano.md#issue_pm_unfused)).
-   - Power **must** be turned off while connecting peripherals.
-   - Improper wiring can lead to *personal harm* or equipment damage!
+> **Warning** The supplied power module is unfused (see [this issue](../flight_controller/cuav_v5_nano.md#issue_pm_unfused)). Power **must** be turned off while connecting peripherals.
```

Comment:
@cuhome OK, I removed the bit about personal harm - Lorenz take on this is that any incorrect wiring anywhere can cause personal harm, so this isn't useful.

However I still left this "is unfused" message - with all the discussion it isn't clear (to me) whether it is or is not unfused.

Reply:
According to the form provided by David; I only saw that DSM / SBUS / RSSI interface PIN1 is unfused; and I have tested the power module again; its fuse is working.

Reply:
So we have to delete it.

Reply:
@davids5 Please confirm this is OK.